addons/website/views/snippets/s_share.xml

```diff
@@ -2,7 +2,7 @@
 <odoo>
 
 <template id="s_share" name="Share">
-    <div t-attf-class="s_share text-left #{_classes}">
+    <div t-attf-class="s_share text-left #{_classes}" data-vxml="001">
```

Comment:
Users who already have a website, use the share block which often acts as a socialMedia block. Instead of doing an upgrade script to modify the existing sites, I think it is easier to display the old share block as deprecated. (even if they really use it as a share block). The worst case scenario for the user is that they were really using the share block as such and they delete it and add it again. However for most users (I think) they will see the message that the block is deprecated and hopefully add the new socialMedia block.